src/shouldHideEvent.js

```diff
@@ -49,6 +49,9 @@ export default function shouldHideEvent(ev) {
     // Hide replacement events since they update the original tile (if enabled)
     if (ev.isRelation("m.replace")) return true;
 
+    // Hide call-related messages based on user preferences
+    if (ev.getType().startsWith("m.call") && !isEnabled('showCallRelatedMessages')) return true;
```

Comment:
There's a chance that we'll have problems with this down the line, so let's make it slightly more specific:
```suggestion
    if (ev.getType().startsWith("m.call.") && !isEnabled('showCallRelatedMessages')) return true;
```

Reply:
Thank you, that's a good idea.